pkg/apis/frameworkcontroller/v1/completion.go

```diff
@@ -193,6 +206,12 @@ func initCompletionCodeInfos() {
 			Type: CompletionType{CompletionTypeNameFailed,
 				[]CompletionTypeAttribute{}},
 		},
+		{
+			Code:   CompletionCodePodCreationUnknownError.Ptr(),
+			Phrase: "PodCreationUnknownError",
+			Type: CompletionType{CompletionTypeNameFailed,
+				[]CompletionTypeAttribute{}},
+		},
```

Comment:
Curious about the behaviors for `CompletionCodePodCreationUnknownError`. Will it be retried? AlwaysRetry or just retry to max?

Reply:
In PAI, it is retry to max